### PR TITLE
#274: Fix date format on contributions table

### PR DESCRIPTION
**Pull request by alexisbarnes**
- Fix the date format on contributor contributions table
- Reorder the table columns, so it's consistent with the other pages
- Remove description column

I think we should move the `Total Contributed` to a summary item at the top right of the page (similar to the candidate page).

Screen Shot 2021-05-22 at 5 46 00 PM

Closes issue #274

### CODE REVIEW

**Review by SchaeStewart — APPROVED** [review]

File: src/hooks/useTableColumns.js [hunk 1]
```diff
@@ -130,6 +130,19 @@ export const useTableColumns = () => {
...
+        Header: 'Donation Date',
+        id: 'date_occurred',
+        accessor: (r) => {
+          const d = new Date(r.date_occurred)
+          return d.toLocaleDateString('en-US', {
+            year: 'numeric',
+            month: 'short',
+            day: '2-digit',
+          })
+        },
```
Comment (mcelestini) [inline src/hooks/useTableColumns.js:143]: this is _really_ nitpicky of me, and you don't need to implement this at all: it's more a trick/preference thing, _but_ you could do this all in one line like so:

```suggestion
        accessor: (r) => new Date(r.date_occurred).toLocaleDateString('en-US', {
            year: 'numeric',
            month: 'short',
            day: '2-digit',
          }),
```

if your function can return in one line, you don't need curly brackets, or even a `return`. Just whatever that line returns is what your function returns

**Comment by mcelestini** [comment]
This is _great_ 👏 . I'm thinking about what you said regarding removing the `Total Contribution` column, but I've got a question about it. If someone contributes to more than one candidate, will it show a different total contribution for each candidate? Or is it always the same number? If it's always the same number, I say we can even remove that column in this PR and move `Total Contribution` to somewhere up top

**Review by mcelestini — APPROVED** [review]
💯

Left a few comments or questions, but they don't block you from merging this if you'd like to. We can address them now or later, up to you 👍

**Comment by alexisbarnes** [comment]
> This is _great_ 👏 . I'm thinking about what you said regarding removing the `Total Contribution` column, but I've got a question about it. If someone contributes to more than one candidate, will it show a different total contribution for each candidate? Or is it always the same number? If it's always the same number, I say we can even remove that column in this PR and move `Total Contribution` to somewhere up top

On the pages I've seen, it's always the same number. Let me double check and go through some more contributor pages before I make the change.

**Review by SchaeStewart — APPROVED** [review]

**Comment by alexisbarnes** [comment]
Made an ugly mess of this branch. I'm going to wait until the expenditures table is merged before trying to push this through. It will definitely need another review.

**Comment by alexisbarnes** [comment]
I've decided not the drop the `Total Contributions` column. It make sense for contributors who have donated to different campaigns. I need to rethink how to handle total contributions for campaigns and overall total contributions. A task for later. Asking for a new review to double check and make sure I didn't accidentally leave unwanted code.

**Review by SchaeStewart — APPROVED** [review]

**Pull request merged**